oddt/fingerprints.py

```diff
@@ -498,23 +503,25 @@ def similaritySPLIF(reference, query, cutoff=1.):
         ref_protein = ref_pair['protein_coords']
         query_ligand = query_pair['ligand_coords']
         query_protein = query_pair['protein_coords']
-        rmsd_ligand = np.sqrt(np.nansum(
-            np.mean(
-                (ref_ligand[:, np.newaxis, ...] - query_ligand)**2, axis=-1), axis=-1))
-        rmsd_protein = np.sqrt(np.nansum(np.mean(
-            (ref_protein[:, np.newaxis, ...] - query_protein)**2, axis=-1), axis=-1))
+        rmsd_ligand = rmsd(ref_ligand - query_ligand)
+        rmsd_protein = rmsd(ref_protein - query_protein)
         n_matching = ((rmsd_ligand < cutoff) & (rmsd_protein < cutoff)).sum()
         numla += n_matching
         numpa += n_matching
         nula += (rmsd_ligand < cutoff).sum()
         nupa += (rmsd_protein < cutoff).sum()
-
     if nula == 0 or nupa == 0:
         return 0.
     else:
         return np.sqrt((numla / nula) * (numpa / nupa))
 
 
+def rmsd(reference, query):
```

Comment:
Move it to the scope of similarity function. It may be misleading, since it does all to all rmsd calculation which is tailored for `similarity_SPLIF`  example. If we need it outside, then we could move it out of this scope.